l10n_es_aeat_mod190/i18n/ca_ES.po

```diff
@@ -0,0 +1,1576 @@
+# Translation of Odoo Server.
```

Comment:
Este archivo no debe existir (debe ser `ca.po`), y menos en los commits de migración.

Reply:
This file is still present and incorrect.